src/main/java/org/apache/commons/cli/CommandLineParser.java

```diff
@@ -36,11 +36,11 @@
      */
     CommandLine parse(Options options, String[] arguments) throws ParseException;
 
-    /**
```

Comment:
Leave as Javadoc since one can specify to Javadoc what level of visibility to document for the generated site.